h/formatters/interfaces.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from zope.interface import Interface
+
+
+class IAnnotationFormatter(Interface):
```

Comment:
I think a top-level comment might be helpful here, explaining what the interface is for as a whole and how it should work. That might be a good place to put the bits of documentation about how the two methods should work together, rather than documenting it separately in each method.